lib/common/util/analytics.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import ReactGA from 'react-ga'
+
+/**
+ * Check if Google Analytics is enabled for the application.
+ */
+export function hasAnalytics (): boolean {
```

Comment:
This can be done written more simply in a variable, right?

```js
const hasAnalytics = process.env.NODE_ENV !== 'development' && process.env.NODE_ENV !== 'test' && !!process.env.GOOGLE_ANALYTICS_TRACKING_ID 
if (!hasAnalytics) console.warn('GOOGLE_ANALYTICS_TRACKING_ID not set')
```